libs/libc/stream/lib_bufferedoutstream.c

```diff
@@ -40,18 +40,18 @@
  * Name: bufferedoutstream_flush
  ****************************************************************************/
 
-static int bufferedoutstream_flush(FAR struct lib_outstream_s *this)
+static int bufferedoutstream_flush(FAR struct lib_outstream_s *self)
 {
-  FAR struct lib_bufferedoutstream_s *rthis =
-    (FAR struct lib_bufferedoutstream_s *)this;
+  FAR struct lib_bufferedoutstream_s *stream =
+    (FAR struct lib_bufferedoutstream_s *)self;
   int ret = OK;
 
-  ret = lib_stream_puts(rthis->backend, rthis->buffer,
-                        rthis->pending);
+  ret = lib_stream_puts(stream->backend, stream->buffer,
+                        stream->pending);
 
   if (ret >= 0)
     {
-      rthis->pending = 0;
+        stream->pending = 0;
```

Comment:
remove the extra indent

Reply:
good catch, my editor added indents here for some reason

Reply:
i've now reviewed my changes and fixed the extra indents in all files I modified, not just the ones you mentioned. good eye!